llama_stack/distribution/datatypes.py

```diff
@@ -176,6 +192,8 @@ class StackRunConfig(BaseModel):
     benchmarks: List[BenchmarkInput] = Field(default_factory=list)
     tool_groups: List[ToolGroupInput] = Field(default_factory=list)
 
+    logger_config: Optional[LoggerConfig] = Field(default=None, description="Configuration of the Llama Stack Logger")
```

Comment:
could the key be called `logging` (since this is nested under a "ServerConfig" structure already)?

Reply:
sure, renamed to logging